tsl/src/data_node_dispatch.c

```diff
@@ -156,16 +156,17 @@ typedef struct DataNodeDispatchState
 	DeparsedInsertStmt stmt; /* Partially deparsed insert statement */
 	const char *sql_stmt;	/* Fully deparsed insert statement */
 	TupleFactory *tupfactory;
-	List *target_attrs;		 /* The attributes to send to remote data nodes */
-	List *responses;		 /* List of responses to process in RETURNING state */
-	HTAB *nodestates;		 /* Hashtable of per-nodestate (tuple stores) */
-	MemoryContext mcxt;		 /* Memory context for per-node state */
-	int64 num_tuples;		 /* Total number of tuples flushed each round */
-	int64 next_tuple;		 /* Next tuple to return to the parent node when in
-							  * RETURNING state */
-	int replication_factor;  /* > 1 if we replicate tuples across data nodes */
-	StmtParams *stmt_params; /* Parameters to send with statement. Format can be binary or text */
-	int flush_threshold;	 /* Batch size used for this dispatch state */
+	List *target_attrs;		  /* The attributes to send to remote data nodes */
+	List *responses;		  /* List of responses to process in RETURNING state */
+	HTAB *nodestates;		  /* Hashtable of per-nodestate (tuple stores) */
+	MemoryContext mcxt;		  /* Memory context for per-node state */
+	MemoryContext batch_mcxt; /* Memory context for batches of data */
+	int64 num_tuples;		  /* Total number of tuples flushed each round */
+	int64 next_tuple;		  /* Next tuple to return to the parent node when in
+							   * RETURNING state */
+	int replication_factor;   /* > 1 if we replicate tuples across data nodes */
+	StmtParams *stmt_params;  /* Parameters to send with statement. Format can be binary or text */
+	int flush_threshold;	  /* Batch size used for this dispatch state */
```

Comment:
Nit: Doesn't seem necessary and also create a lot of different indents for the comments (one more than the one before and one less than the one after).

Reply:
This is `make format`'s doing, so is needed to pass formatting checks.